### PR TITLE
Basic idea forms

### PR DESCRIPTION
**Pull request by 2e2a**
Allows to create, edit and delete ideas.

### CODE REVIEW

File: meinberlin/templates/base.html [hunk 1]
```diff
@@ -28,6 +28,11 @@
         </div>
     </header>
     <main>
+        {% for message in messages %}
+            <li {% if message.tags %} class="{{ message.tags }}"{% endif %}>
+                {{ message }}
+            </li>
+        {% endfor %}
```
Comment (xi) [inline meinberlin/templates/base.html]: `<ul>` is missing

Comment (2e2a) [inline meinberlin/templates/base.html]: pushed a fix

File: meinberlin/settings/base.py [hunk 2]
```diff
@@ -189,6 +189,7 @@
 
 # ckeditor
 
+CKEDITOR_JQUERY_URL = 'https://code.jquery.com/jquery-3.1.1.min.js'
```
Comment (xi) [inline meinberlin/settings/base.py]: We have a jquery bundled in `vendors.js`

File: apps/ideas/templates/meinberlin_ideas/idea_detail.html [hunk 3]
```diff
@@ -32,4 +32,11 @@
         {% react_ratings idea %}
     </div>
 </div>
+
+{% has_perm 'meinberlin_ideas.update_idea' request.user idea as update_allowed %}
+{% if update_allowed %}
+    <a href="{% url 'idea-update' slug=idea.slug %}">
+        {% trans 'Edit idea' %}
+    </a>
+{% endif %}
```
Comment (xi) [inline apps/ideas/templates/meinberlin_ideas/idea_detail.html]: Should be integrated into the dropdown

**Pull request merged**